aws_lambda_builders/workflows/python_pip/workflow.py

```diff
@@ -65,8 +66,19 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
 
         self.actions = [
             PythonPipBuildAction(artifacts_dir, scratch_dir, manifest_path, runtime, binaries=self.binaries),
-            CopySourceAction(source_dir, artifacts_dir, excludes=self.EXCLUDED_FILES),
+            CopySourceAction(source_dir, artifacts_dir, excludes=self.excluded_files),
         ]
 
+    @property
+    def excluded_files(self):
+        """
+        Use .samignore falling back to default exclude files
+        """
+        sam_ignore = pathlib.Path("./.samignore")
```

Comment:
.samignore is a new directory for users to have files that are excluded from the build correct?

Reply:
It's actually a file, akin to `.dockerignore`, `.gitignore` and the like. Which users could place sam specific files/directories to ignore